### ISSUE TITLE
Add Videos to Magento Demo

### ISSUE THREAD

**Issue by rain2o** [DivanteLtd/vue-storefront#2436]
## What is the motivation for adding / enhancing this feature?
In order to demo and test the ability to have videos in product gallery from #2388 and PR #2433 would it be possible to have some videos added to the VS Demo Magento shop? Adding at least 1 from YouTube and Vimeo each would be great for testing and demoing. 

## What are the acceptance criteria 
- At least 1 video added to a product gallery from YouTube and from Vimeo

## Can you complete this feature request by yourself?
No

**Comment by patzick** [comment]
This should be done if #2433 will be merged to develop.

**Comment by pkarw** [comment]
@rain2o please contact me on peter@vuestorefront.io so I could send you magento demo admin access data

**Comment by patzick** [comment]
@rain2o did you managed to add these videos?
Please provide products to test before closing it:)

**Comment by rain2o** [comment]
@patzick Not yet. I emailed @pkarw to request the credentials, just waiting for those.

**Comment by rain2o** [comment]
Also [mage2vuestorefront#71](https://github.com/DivanteLtd/mage2vuestorefront/issues/71) will need to be implemented in order to import the video data.

**Comment by rain2o** [comment]
I have added a video from each source on the Luma Yoga For Life product (http://demo-magento2.vuestorefront.io/luma-yoga-for-life.html & https://demo.vuestorefront.io/p/240-LV09/luma-yoga-for-life). I tried to find sample videos of the products but didn't have a lot of luck so used a VS demo video for YouTube. 

The Vimeo one isn't loading when testing on the Luma theme, but I think it's because the Magento demo isn't using SSL, because it should work with no configuration. Should work once it gets ported to VS. 

The video data can be pumped into VS using the changes from [mage2vuestorefront#75](https://github.com/DivanteLtd/mage2vuestorefront/pull/75).
